src/utils/fs.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Checks files exist
+ * @param files the path to files to check
+ * @returns {boolean}
+ */
+const filesExist = (...files) => files.map(fs.existsSync).every((resp) => resp);
```

Comment:
what is the `.every((resp) => resp)` doing?

Reply:
I'll change it to `Boolean`